src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/CourseController.kt

```diff
@@ -134,4 +135,20 @@ constructor(
 
     return ResponseEntity.ok(savedCourse.toApi())
   }
+
+  override fun createCourse(courseCreateRequest: CourseCreateRequest): ResponseEntity<Course> {
+    val course = CourseEntity(
+      name = courseCreateRequest.name,
+      identifier = courseCreateRequest.identifier,
```

Comment:
I think the course identifier needs some validation - check that there isn't already a course in the database with this identifier